bindings/nodejs/examples/how_tos/accounts_and_addresses/create-wallet.ts

```diff
@@ -55,7 +55,7 @@ async function run() {
         });
 
         const walletOptions: WalletOptions = {
-            address: wallet_address[0],
+            address: walletAddress[0],
```

Comment:
Maybe put the [0] in the declaration already, bit weird otherwise

Reply:
But needs extra () then because the generation is async which also looks weird

Reply:
Ah, I see